app/static/js/map.js

```diff
@@ -263,25 +271,40 @@ const displayResult = function (map, pixel) {
     }
 };
 
-export function initializeMaps() {
+const $mapContainer = $(".map__container");
+let $arcticScreenReaderList = $("#map__screenreader--arctic");
+let $antarcticScreenReaderList = $("#map__screenreader--antarctic");
+
+
+export function initializeMaps(lazy=false) {
+    $arcticScreenReaderList = $("#map__screenreader--arctic");
+    $antarcticScreenReaderList = $("#map__screenreader--antarctic");
+
+    $arcticScreenReaderList.empty();
+    $antarcticScreenReaderList.empty();
+
     arcticResultsSource.clear(true);
     antarcticResultsSource.clear(true);
 
-    $(".map__container").removeClass("hidden");
+    $mapContainer.removeClass("hidden");
 
-    if (!arcticMap) {
+
+    if (!lazy || !arcticMap) {
```

Comment:
This function can now initialize a new map each time it's called, or only initialize a map if there isn't one already (lazy initialization)